knot-thing/src/main/java/com/cesar/knot_sdk/knot_state_machine/KNoTStateMachine.kt

```diff
@@ -0,0 +1,61 @@
+
+package com.cesar.knot_sdk.knot_state_machine
+
+import android.content.Context
+import com.cesar.knot_sdk.KNoTAMQP
+import com.cesar.knot_sdk.KNoTDataManager
+import com.cesar.knot_sdk.KNoTMessageParser
+import com.cesar.knot_sdk.KNoTMessager
+import com.cesar.knot_sdk.knot_state_machine.states.base_classes.State
+import com.cesar.knot_sdk.knot_state_machine.states.base_classes.branching_state.BranchingState
+import com.cesar.knot_sdk.knot_state_machine.states.base_classes.procedural_state.ProceduralState
+import com.cesar.knot_sdk.knot_state_machine.states.base_classes.waiting_state.WaitingState
+
+object KNoTStateMachine {
+
+    val DEFAULT_RABBIT_PORT_NUMBER = 5672
+    val HOSTNAME = ""
+    val PORT_NUMBER = DEFAULT_RABBIT_PORT_NUMBER
+    val USERNAME = ""
+    val PASSWORD = ""
```

Comment:
Please, add a note specifying the developer should rewrite these constants for hardcode values. Also, I think you can use the `'const'` keyword on these values.